src/World.h

```diff
@@ -654,7 +654,7 @@ class cWorld  // tolua_export
 
 	/** Spawns a new ender crystal at the specified block coords.
 	Returns the UniqueID of the created entity, or cEntity::INVALID_ID on failure. */
-	UInt32 SpawnEnderCrystal(Vector3d a_Pos, bool a_ShowBottom = false);
+	UInt32 SpawnEnderCrystal(Vector3d a_Pos, bool a_ShowBottom, Vector3i a_BeamTarget, bool a_DisplayBeam);
```

Comment:
This will break any users of the lua API, right?

Reply:
Yes, but I added the function yesterday so I don't think anyone is using it

Reply:
Ah, okay then